### PR TITLE
Refactor skip_grid, skip_save. Change default to webp. Add '@' toggle

### PR DESCRIPTION
**Pull request by jjisnow**
I'm new at PR, so apologies if it's messy. 

* refactored skip_grid and skip_save into variables that don't require thinking in double negatives when reading code
* webp:65 grid format and default image format makes for smaller file sizes with minimal appreciable visual quality loss
* '@' without a prompt matrix seems like a useful way to instruct the code not to save or display the original file

### CODE REVIEW

File: webui.py [hunk 1]
```diff
@@ -1959,9 +1965,9 @@ def run_RealESRGAN(image, model_name: str):
 
 txt2img_defaults = {
     'prompt': '',
-    'ddim_steps': 50,
+    'ddim_steps': 25,
     'toggles': [1, 2, 3],
-    'sampler_name': 'k_lms',
+    'sampler_name': 'k_euler_a',
```
Comment (JohannesGaessler) [inline webui.py:1970]: Do not add random changes that are not documented in the description of your PR.
Did you perhaps commit your own modifications by accident?

File: webui.py [hunk 2]
```diff
@@ -2030,9 +2036,9 @@ def run_RealESRGAN(image, model_name: str):
 
 img2img_defaults = {
     'prompt': '',
-    'ddim_steps': 50,
+    'ddim_steps': 25,
     'toggles': [1, 4, 5],
-    'sampler_name': 'k_lms',
+    'sampler_name': 'k_euler_a',
```
Comment (JohannesGaessler) [inline webui.py:2041]: Same as above, do not add undocumented changes.

**Review by JohannesGaessler — COMMENTED** [review]
>default image format 

This is not the default this is hard-coded (for individual images).
It's a nitpick but it matters because you cannot set image formats via config.

In general I think that image formats should not be changed to webp since 4chan still does not support it (I think).

Also @hlky , if you decide to merge, please squash beforehand to avoid spamming the git log.

**Comment by codedealer** [comment]
Changing the default format to webp is too unfriendly a feature to just throw in there. I think it needs to be discussed separately.

Can you even open a webp on windows 10 without 3rd party tools? It makes for great lossless compression no doubt but it's also Chrome centric.

File: webui.py (hunk 2 above)
Comment (jjisnow) [reply webui.py:2041]: Good point. These are defaults of my own convenience and I have yet to figure out a way of breaking up my PR and squashing down my commit tree.

Comment (JohannesGaessler) [reply webui.py:2041]: I recommend using a config file to change the defaults.
That way your change in parameters is completely decoupled from any changes to code.

Alternatively use git to stash your private changes, do any dev work that you want to do, commit your changes, and apply the stashed private changes.

Comment (jjisnow) [reply webui.py]: Thanks for the tip.

**Comment by jjisnow** [comment]
In trying to clean up the PR i believe I mangled the remote commit tree.

**Pull request closed without merging**